modules/lightmapper_rd/lm_blendseams.glsl

```diff
@@ -0,0 +1,117 @@
+/* clang-format off */
+[versions]
+
+lines = "#define MODE_LINES"
```

Comment:
its the same syntax as the new glsl resources that can be imported, in this case I can compile them on the fly in the lightmapper since this uses rendering device on its own and not via rendering server.